src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -111,22 +114,20 @@ public void reload() {
             String elbName = checkLoadBalancerName(LOAD_BALANCER_NAME);
             Set<String> targetAddresses = super.getServerPoolAddresses();
 
-            LoadBalancerServiceContext loadBalancerServiceContext = null;
             try {
                 loadBalancerServiceContext = newLoadBalancerServiceContext(loc.getIdentity(), loc.getCredential());
-                ELBApi api = api(loadBalancerServiceContext);
 
                 Set<String> instances = ImmutableSet.copyOf(targetAddresses);
-                LoadBalancer loadBalancer = api.getLoadBalancerApi().get(elbName);
+                LoadBalancer loadBalancer = getELBApi().getLoadBalancerApi().get(elbName);
```

Comment:
good point, if i change the elbapi back to a local variable (as in the comment above) it will address this